src/test/regress/expected/view_propagation.out

```diff
@@ -411,7 +411,7 @@ DROP TABLE view_table_2 CASCADE;
 NOTICE:  drop cascades to 2 other objects
 DETAIL:  drop cascades to view prop_view_2
 drop cascades to constraint f_key_for_local_table on table view_table_3
-NOTICE:  drop cascades to constraint f_key_for_local_table_1410200 on table view_prop_schema.view_table_3_1410200
+NOTICE:  drop cascades to constraint f_key_for_local_table_1410201 on table view_prop_schema.view_table_3_1410201
```

Comment:
I wonder why this changed.

Reply:
looks like we don't have a `SET next_shard_id` command in `view_propagation` so it's affected by some other test file

Reply:
Let's add a set next_shard_id in that test instead then